tests/test_HIP_RA.py

```diff
@@ -1,36 +1,25 @@
-import os.path
-import sys
-import unittest
 from pathlib import Path
 
-from geophires_x import HIP_RA
+from base_test_case import BaseTestCase
+from hip_ra import HipRaClient
+from hip_ra import HipRaInputParameters
 
 
 # noinspection PyTypeChecker
-class HIP_RATestCase(unittest.TestCase):
-    maxDiff = None
-
+class HIP_RATestCase(BaseTestCase):
     def test_HIP_RA_examples(self):
         example_files = self._list_test_files_dir(test_files_dir='examples')
 
-        def get_output_file_for_example(example_file: str):
-            return self._get_test_file_path(
-                Path('examples', f'{example_file.split(".txt")[0].capitalize()}V3_output.txt')
-            )
+        client = HipRaClient()
+
+        def get_output_file_for_example(example_file: Path):
+            return self._get_test_file_path(Path(example_file).with_suffix('.out'))
 
         for example_file_path in example_files:
-            if example_file_path.startswith('HIPexample') and '_output' not in example_file_path:
```

Comment:
This is the line that was filtering `example1_outputunits` which led to my mistakenly resolution of https://github.com/NREL/python-geophires-x/issues/11 as described in https://github.com/NREL/python-geophires-x/issues/18